specs-go/config.go

```diff
@@ -117,6 +117,11 @@ type Mount struct {
 	Source string `json:"source,omitempty"`
 	// Options are fstab style mount options.
 	Options []string `json:"options,omitempty"`
```

Comment:
The idmap option is one of the fstab options and that is why we don't need to mention it in any other part of the spec?

Reply:
I guess it is explicit when a mapping is specified?

Reply:
Oh, so we use idmap if a mapping is specified? Makes sense. Thanks!